playbooks/streisand-client.yml

```diff
@@ -0,0 +1,20 @@
+---
```

Comment:
Should we name these new playbook files something else? Maybe `vagrant-testing-streisand-client.yml` is too ugly/long, but I could see the current filename causing some confusion for people who run Streisand on non-native providers outside of the wrapper script--especially when tab complete will suggest this as a possibility given its similarity to `streisand.yml`.